docker-compose.video.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
+
+services:
+  chrome_video:
+    container_name: "${PROJECT_NAME}_chrome_video"
+    image: selenium/video:${SELENIUM_FFMPEG_TAG}
+    depends_on:
+      - chrome
+    environment:
+      - DISPLAY_CONTAINER_NAME=chrome
+      - FILE_NAME=chrome_video.mp4
```

Comment:
@mistermoper How do I get the video? I do not see a volume shared with the real machine. Do I need a docker cp or something similar?